pkg/apis/options/provideropts/providers.go

```diff
@@ -1,4 +1,6 @@
-package options
+package provideropts
```

Comment:
Why is this moving?

Reply:
due to import cycles, as described [here](https://github.com/oauth2-proxy/oauth2-proxy/pull/1923#discussion_r1060519964),

Reply:
resolved. moved to original location.